src/ex4/server/routes/tasksApi.js

```diff
@@ -13,17 +13,28 @@ router.get("/", async (req, res) => {
 router.post("/", async (req, res) => {
   const taskInput = req.body.itemName;
   const isCompleted = req.body.status;
-  const isAdded = await itemManager.addTask(taskInput, isCompleted);
-  if (isAdded) {
-    res.status(200).json(req.body);
+  const position = req.body.position;
+  const response = await itemManager.addTask(taskInput, isCompleted, position);
+  if (response) {
+    res.status(200).json(response);
   } else {
     res.status(409).json({ error: "Task already exists" });
   }
 });
 
+router.put("/resort", (req, res) => {
+  const sorted = itemManager.reSortTasks(req.body);
```

Comment:
sending the entire body is a bad practice, you always want to know what should be in the body in the controller level, also people can send anything they want in the body so it's dangerous to just pass it as-is